middlewares/stripPrefixRegex_test.go

```diff
@@ -73,18 +73,16 @@ func TestStripPrefixRegex(t *testing.T) {
 				actualPath = r.URL.Path
 				actualHeader = r.Header.Get(ForwardedPrefixHeader)
 			})
-
 			handler := NewStripPrefixRegex(handlerPath, testPrefixRegex)
-			server := httptest.NewServer(handler)
-			defer server.Close()
 
-			resp, err := http.Get(server.URL + test.path)
-			require.NoError(t, err, "%s: unexpected error.", test.path)
+			req := testhelpers.MustNewRequest(http.MethodGet, "http://localhost"+test.path, nil)
+			resp := &httptest.ResponseRecorder{Code: http.StatusOK}
+
+			handler.ServeHTTP(resp, req)
 
-			assert.Equal(t, test.expectedStatusCode, resp.StatusCode, "%s: unexpected status code.", test.path)
+			assert.Equal(t, test.expectedStatusCode, resp.Code, "%s: unexpected status code.", test.path)
```

Comment:
Hmm, I'd rather not pollute our tests with duplicate identifiers due to one Go IDE being buggy. On all of Visual Studio Code, Atom, and vim-go, this is not a problem. Might it be acceptable to switch to the terminal and run `go test` manually for cases where tests fail on Gogland until this gets fixed?

You pasted a go test text output. Looking at [this screenshot from a Gogland issue](https://youtrack.jetbrains.com/issue/GO-2957#u=1481990264537&viewimage=snapshot210.png%3Ffile=74-370189&c=true&rw=1574&rh=432&u=1481990264537), it looks like the IDE does show the sub-test name on the left pane?